middleware/TelemetryHandler.md

```diff
@@ -36,6 +36,10 @@ Provide a mandatory middleware component that attaches metadata to a Graph reque
     - `RuntimeEnvironment: Node/1.1` for JavaScript.
     - `RuntimeEnvironment: JRE/1.1` for Java.
 
+- Certain workloads error out when an unexpected header is present in the request. The middlewares should check:
```

Comment:
Do we have workloads that have a problem with the Telemetry headers? 
We have mentioned the Auth header before as an example with the Outlook large file upload. However, my understanding is that might be a special case since we are trying to authenticate in a situation that is unnecessary to authenticate (the url is unique and will eventually expire). 
In such a scenario wouldn't we want to know if the uploader was using the SDK or not to make the upload?

Reply:
AFAIK servers are supposed to ignore HTTP request headers they don't understand, but I'll defer to @darrelmiller on this.
Assuming I'm correct on that aspect, I don't think we need to delete the custom SDK version header, it can be useful for telemetry and debugging should anything go wrong, and if workloads don't care about it, they can simply ignore it.

Reply:
https://github.com/microsoftgraph/msgraph-sdk-javascript/issues/265

This is an example of  LargeFileUploadTask upload to OneDrive caused CORS issue due sdkversion telemetry header